### PR TITLE
Edits for sections 0 to 2.2

### PR DESCRIPTION
**Pull request by pllim**
My edits from sections 0 (beginning) to 2.2. I don't have time to finish the paper in one day, so edits for subsequent sections will come as separate pull request(s).

### CODE REVIEW

File: main.tex [hunk 1]
```diff
@@ -113,40 +113,40 @@ \section{Introduction} \label{sec:intro}
...
 
-The \astropy project aims to develop and provide high quality code and
+The \astropy project aims to develop and provide high-quality code and
 documentation according to the best practices in software development.
 The project makes use of these tools to do so without central institutional
 oversight.
 The first public release of the \astropypkg package is described in
 \cite{astropy}. Since then, the \astropypkg package has been
-used in hundreds of projects, and the scope of the package has grown
-considerably. At the same time, the community of astronomers
```
Comment (pllim) [inline main.tex:149]: NOTE: Not all contributors are strictly astronomers.

File: main.tex [hunk 2]
```diff
@@ -198,7 +198,7 @@ \subsection{Astropy development model}
 with a set of possible labels that help classify and organize the issues.
 The development workflow is detailed in the \astropypkg
 documentation.\footnote{\emph{How to make a code contribution},
-\url{http://docs.astropy.org/en/stable/development/workflow/development_workflow.html}}
+\url{http://docs.astropy.org/en/latest/development/workflow/development_workflow.html}}
```
Comment (pllim) [inline main.tex:201]: NOTE: Since contributors submit changes against `dev`, it makes more sense to point to the `dev` version of the doc.

Comment (bsipocz) [reply main.tex:201]: I had a point earlier to point to the stable docs, but agree with @pllim here that the dev docs should point to the latest versions not stable.

File: main.tex (hunk 1 above)
Comment (bsipocz) [reply main.tex:149]: very much agree with this, thanks!

**Review by bsipocz — APPROVED** [review]
Looks good to me, but I leave merging to a native speaker.

**Review by hamogu — APPROVED** [review]

**Comment by pllim** [comment]
Thank you!

**Pull request merged**